commit 91a1095fc760d625cd36e0a840f76a3a21530230
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Tue Feb 7 22:37:11 2017 -0500

    fix doc/Makefile.am

diff --git a/docs/Makefile.am b/docs/Makefile.am
index 5fd3fc1..da65d78 100644
--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -36,7 +36,8 @@ content_files =		\
 	version.xml
 HTML_IMAGES =
 
-include $(top_srcdir)/gtk-doc.make
+#include $(top_srcdir)/gtk-doc.make
+EXTRA_DIST=""
 
 # Other files to distribute
 EXTRA_DIST += version.xml.in
