diff --git a/create-dmg b/create-dmg
index fcdf02a..7354973 100755
--- a/create-dmg
+++ b/create-dmg
@@ -267,7 +267,7 @@ if ! test -z "$ADD_FOLDER_SOURCES"; then
 fi
 
 # Add extra space for additional resources
-DISK_IMAGE_SIZE=$(expr $DISK_IMAGE_SIZE + 20)
+DISK_IMAGE_SIZE=$(expr $DISK_IMAGE_SIZE + 150)
 
 # Resize the image for the extra stuff
 hdiutil resize ${HDIUTIL_VERBOSITY} -size ${DISK_IMAGE_SIZE}m "${DMG_TEMP_NAME}"
