diff --git a/gr-video-sdl/CMakeLists.txt b/gr-video-sdl/CMakeLists.txt
index 50975eb67..6de4a15ec 100644
--- a/gr-video-sdl/CMakeLists.txt
+++ b/gr-video-sdl/CMakeLists.txt
@@ -22,14 +22,14 @@
 ########################################################################
 include(GrBoost)
 
-find_package(SDL)
+find_package(SDL2)
 
 ########################################################################
 # Register component
 ########################################################################
 include(GrComponent)
 GR_REGISTER_COMPONENT("gr-video-sdl" ENABLE_GR_VIDEO_SDL
-    SDL_FOUND
+    SDL2_FOUND
     Boost_FOUND
     ENABLE_GNURADIO_RUNTIME
 )
