commit 3ad55192ba3ae56c5e372fe5dfdd5d43ec8f64de
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Wed Feb 8 22:01:39 2017 -0500

    add _Nonnull and _Nullable to lexer ignore primitives

diff --git a/giscanner/scannerlexer.l b/giscanner/scannerlexer.l
index 941154d..fec86a9 100644
--- a/giscanner/scannerlexer.l
+++ b/giscanner/scannerlexer.l
@@ -164,6 +164,9 @@ stringtext				([^\\\"])|(\\.)
 "__inline"				{ return INLINE; }
 "__nonnull" 			        { if (!parse_ignored_macro()) REJECT; }
 "_Noreturn" 			        { /* Ignore */ }
+"_Nonnull" 			        { /* Ignore */ }
+"_Nullable" 			        { /* Ignore */ }
+"nullable" 			        { /* Ignore */ }
 "__signed__"				{ return SIGNED; }
 "__restrict"				{ return RESTRICT; }
 "__restrict__"				{ return RESTRICT; }
