commit 39cda218e6b717525c84fe6414917abc47aac619
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Tue Feb 14 05:29:24 2017 -0500

    added BASE_DEPEDENCIES_CFLAGS to gtk/Makefile.am

diff --git a/gtk/Makefile.am b/gtk/Makefile.am
index 8cc4536..33cb720 100644
--- a/gtk/Makefile.am
+++ b/gtk/Makefile.am
@@ -58,6 +58,7 @@ INCLUDES =						\
 	-I$(top_builddir)/gtk				\
 	-I$(top_srcdir) -I../gdk			\
 	-I$(top_srcdir)/gdk				\
+        $(BASE_DEPENDENCIES_CFLAGS)			\
 	$(GMODULE_CFLAGS)				\
 	$(GTK_DEBUG_FLAGS)				\
 	$(GTK_DEP_CFLAGS)				\
