diff --git a/gtk/a11y/gtkaccessibility.c b/gtk/a11y/gtkaccessibility.c
index 7f0e520..d3ca029 100644
--- a/gtk/a11y/gtkaccessibility.c
+++ b/gtk/a11y/gtkaccessibility.c
@@ -37,9 +37,6 @@
 #include <gtk/gtktogglebutton.h>
 #include <gtk/gtkaccessible.h>
 
-#ifdef GDK_WINDOWING_X11
-#include <atk-bridge.h>
-#endif
 
 static gboolean gail_focus_watcher      (GSignalInvocationHint *ihint,
                                          guint                  n_param_values,
@@ -988,9 +985,5 @@ _gtk_accessibility_init (void)
   _gtk_accessibility_override_atk_util ();
   do_window_event_initialization ();
 
-#ifdef GDK_WINDOWING_X11
-  atk_bridge_adaptor_init (NULL, NULL);
-#endif
-
   atk_misc_instance = g_object_new (GTK_TYPE_MISC_IMPL, NULL);
 }
diff --git a/meson.build b/meson.build
index 6d6a4d1..9ef0a0b 100644
--- a/meson.build
+++ b/meson.build
@@ -126,10 +126,10 @@ gtk_osxversions = [osx_current, '@0@.@1@.0'.format(osx_current, gtk_interface_ag
 
 gtk_api_version = '@0@.0'.format(gtk_major_version)
 
-x11_enabled            = get_option('x11_backend')
+x11_enabled            = true
 wayland_enabled        = get_option('wayland_backend')
 broadway_enabled       = get_option('broadway_backend')
-quartz_enabled         = get_option('quartz_backend')
+quartz_enabled         = false
 win32_enabled          = get_option('win32_backend')
 mir_enabled            = get_option('mir_backend')
 
@@ -153,7 +153,6 @@ os_unix = not os_win32
 
 if os_darwin
   wayland_enabled = false
-  x11_enabled = false
 else
   quartz_enabled = false
 endif
@@ -562,7 +561,6 @@ if x11_enabled
   xfixes_dep     = dependency('xfixes', required: false)
   xcomposite_dep = dependency('xcomposite', required: false)
   fontconfig_dep = dependency('fontconfig', fallback: ['fontconfig', 'fontconfig_dep'])
-  atkbridge_dep  = dependency('atk-bridge-2.0', version: atk_req)
 
   x11_pkgs = ['fontconfig', 'x11', 'xext', 'xi', 'xrandr']
 
@@ -579,7 +577,6 @@ if x11_enabled
     x11_pkgs += ['xdamage']
   endif
 
-  atk_pkgs += ['atk-bridge-2.0']
 
   cdata.set('HAVE_XDAMAGE', xdamage_dep.found() ? 1 : false)
   cdata.set('HAVE_XCURSOR', xcursor_dep.found() ? 1 : false)
