commit 0d79b1bef93a3a3591ee8b6a18f4997f2379c6a9
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Wed Feb 8 19:22:19 2017 -0500

    remove references for gtk-doc.make

diff --git a/configure.ac b/configure.ac
index 4e11b40..3db7d67 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1595,7 +1595,7 @@ GOBJECT_INTROSPECTION_CHECK([0.9.3])
 # Checks for gtk-doc and docbook-tools
 ##################################################
 
-GTK_DOC_CHECK([1.11])
+#GTK_DOC_CHECK([1.11])
 
 AC_CHECK_PROG(DB2HTML, db2html, true, false)
 AM_CONDITIONAL(HAVE_DOCBOOK, $DB2HTML)
diff --git a/docs/reference/gdk/Makefile.am b/docs/reference/gdk/Makefile.am
index 5704c69..64f4118 100644
--- a/docs/reference/gdk/Makefile.am
+++ b/docs/reference/gdk/Makefile.am
@@ -148,7 +148,8 @@ FIXXREF_OPTIONS= \
 	--extra-dir=$(CAIRO_PREFIX)/share/gtk-doc/html/cairo
 
 
-include $(top_srcdir)/gtk-doc.make
+#include $(top_srcdir)/gtk-doc.make
+EXTRA_DIST=
 
 # Other files to distribute
 EXTRA_DIST += version.xml.in
diff --git a/docs/reference/gtk/Makefile.am b/docs/reference/gtk/Makefile.am
index c239445..8a48f00 100644
--- a/docs/reference/gtk/Makefile.am
+++ b/docs/reference/gtk/Makefile.am
@@ -359,7 +359,8 @@ FIXXREF_OPTIONS=--extra-dir=../gdk/html \
 	--extra-dir=$(PANGO_PREFIX)/share/gtk-doc/html/pango \
 	--extra-dir=$(CAIRO_PREFIX)/share/gtk-doc/html/cairo
 
-include $(top_srcdir)/gtk-doc.make
+#include $(top_srcdir)/gtk-doc.make
+EXTRA_DIST=
 
 # Other files to distribute
 EXTRA_DIST += version.xml.in
diff --git a/docs/reference/libgail-util/Makefile.am b/docs/reference/libgail-util/Makefile.am
index e6b4bb9..7b6473b 100644
--- a/docs/reference/libgail-util/Makefile.am
+++ b/docs/reference/libgail-util/Makefile.am
@@ -31,6 +31,7 @@ content_files =
 
 HTML_IMAGES =
 
-include $(top_srcdir)/gtk-doc.make
+#include $(top_srcdir)/gtk-doc.make
+EXTRA_DIST=
 
 -include $(top_srcdir)/git.mk
