commit 24d8301d7a878bb5710dbcc2cba7ab614a58fda8
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Mon Feb 20 14:58:08 2017 -0500

    do not build gtk-doc

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 5db7716..5518d2a 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -125,7 +125,8 @@ GTKDOC_LIBS = \
 	$(GTK_LIBS)
 
 # This includes the standard gtk-doc make rules, copied by gtkdocize.
-include $(top_srcdir)/gtk-doc.make
+# include $(top_srcdir)/gtk-doc.make
+EXTRA_DIST=""
 
 # Other files to distribute
 # e.g. EXTRA_DIST += version.xml.in
@@ -138,9 +139,9 @@ EXTRA_DIST += \
 #DISTCLEANFILES +=
 
 # Comment this out if you want your docs-status tested during 'make check'
-if ENABLE_GTK_DOC
+#if ENABLE_GTK_DOC
 #TESTS_ENVIRONMENT = cd $(srcsrc) &&
 #TESTS = $(GTKDOC_CHECK)
-endif
+#endif
 
 -include $(top_srcdir)/git.mk
