diff --git a/test/rmlvo-to-keymap.c b/test/rmlvo-to-keymap.c
index ed48e49..8e7bef3 100644
--- a/test/rmlvo-to-keymap.c
+++ b/test/rmlvo-to-keymap.c
@@ -36,7 +36,7 @@ static void
 usage(void)
 {
     printf("Usage: %s [--print] [--rules <rules>] [--layout <layout>] [--variant <variant>] [--options <option>]\n",
-           program_invocation_short_name);
+           getprogname());
     printf("This tool tests the compilation from RMLVO to a keymap.\n");
     printf("--print  print the resulting keymap\n");
 }
