commit 76540c5fd1ae464b607f1448fc2ddc0bb3c757c4
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Wed Feb 8 22:35:50 2017 -0500

    fix GLYPH_EMPTY bullshit

diff --git a/pango/pango-font.h b/pango/pango-font.h
index cdc07cd..6d159a1 100644
--- a/pango/pango-font.h
+++ b/pango/pango-font.h
@@ -674,7 +674,10 @@ struct _PangoFontClass
  *
  * Returns: a #PangoGlyph value that means no glyph was found for @wc.
  */
-#define PANGO_GLYPH_EMPTY           ((PangoGlyph)0x0FFFFFFF)
+enum {
+PANGO_GLYPH_EMPTY = ((PangoGlyph)0x0FFFFFFF)
+};
+//#define PANGO_GLYPH_EMPTY           ((PangoGlyph)0x0FFFFFFF)
 #define PANGO_GLYPH_INVALID_INPUT   ((PangoGlyph)0xFFFFFFFF)
 #define PANGO_GLYPH_UNKNOWN_FLAG    ((PangoGlyph)0x10000000)
 #define PANGO_GET_UNKNOWN_GLYPH(wc) ((PangoGlyph)(wc)|PANGO_GLYPH_UNKNOWN_FLAG)
