commit bc569409c4d23fb95ee10ca79ab4ef1f1f963732
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Sun Feb 5 21:26:56 2017 -0500

    fixed usage of ERROR_DOMAIN in pygi-info.c

diff --git a/gi/pygi-info.c b/gi/pygi-info.c
index 1bfd7d8..271cfb1 100644
--- a/gi/pygi-info.c
+++ b/gi/pygi-info.c
@@ -162,9 +162,12 @@ _pygi_info_new (GIBaseInfo *info)
         case GI_INFO_TYPE_CONSTANT:
             type = &PyGIConstantInfo_Type;
             break;
+/*
         case GI_INFO_TYPE_ERROR_DOMAIN:
             type = &PyGIErrorDomainInfo_Type;
             break;
+*/
+
         case GI_INFO_TYPE_UNION:
             type = &PyGIUnionInfo_Type;
             break;
@@ -481,7 +484,7 @@ _pygi_g_type_info_size (GITypeInfo *type_info)
                 case GI_INFO_TYPE_INVALID:
                 case GI_INFO_TYPE_FUNCTION:
                 case GI_INFO_TYPE_CONSTANT:
-                case GI_INFO_TYPE_ERROR_DOMAIN:
+//                case GI_INFO_TYPE_ERROR_DOMAIN:
                 case GI_INFO_TYPE_VALUE:
                 case GI_INFO_TYPE_SIGNAL:
                 case GI_INFO_TYPE_PROPERTY:
@@ -860,7 +863,7 @@ pygi_g_struct_info_is_simple (GIStructInfo *struct_info)
                     case GI_INFO_TYPE_INVALID:
                     case GI_INFO_TYPE_FUNCTION:
                     case GI_INFO_TYPE_CONSTANT:
-                    case GI_INFO_TYPE_ERROR_DOMAIN:
+//                    case GI_INFO_TYPE_ERROR_DOMAIN:
                     case GI_INFO_TYPE_VALUE:
                     case GI_INFO_TYPE_SIGNAL:
                     case GI_INFO_TYPE_PROPERTY:
