diff --git a/sip/QtGui/qabstractprintdialog.sip b/sip/QtGui/qabstractprintdialog.sip
index 4389370..90eb98a 100644
--- a/sip/QtGui/qabstractprintdialog.sip
+++ b/sip/QtGui/qabstractprintdialog.sip
@@ -54,9 +54,6 @@ public:
 %If (Qt_4_4_0 -)
         DontUseSheet,
 %End
-%If (Qt_4_7_0 -)
-        PrintCurrentPage,
-%End
     };
 
 %End
@@ -73,7 +70,6 @@ enum PrintDialogOption {
     PrintPageRange,
     PrintCollateCopies,
     PrintShowPageSize,
-    PrintCurrentPage
 };
 %End
     typedef QFlags<QAbstractPrintDialog::PrintDialogOption> PrintDialogOptions;
