diff --git a/src/opengl/Makefile.Release b/src/opengl/Makefile.Release
index 71f88de..547ce60 100644
--- a/src/opengl/Makefile.Release
+++ b/src/opengl/Makefile.Release
@@ -8,7 +8,7 @@
 ####### Compiler, tools and options
 
 CC            = clang -mmacosx-version-min=10.7 -I/Applications/GNURadio.app/Contents/MacOS/usr/include -I/opt/X11/include
-CXX           = clang++ -stdlib=libc++ -mmacosx-version-min=10.7 -I/Applications/GNURadio.app/Contents/MacOS/usr/include -I/opt/X11/include
+CXX           = clang++ -stdlib=libc++ -mmacosx-version-min=10.7 -I/Applications/GNURadio.app/Contents/MacOS/usr/include -I/opt/X11/include -I/System/Library/Frameworks/OpenGL.framework/Versions/A/Headers
 DEFINES       = -DQT_SHARED -DQT_BUILD_OPENGL_LIB -DQT_NO_USING_NAMESPACE -DQT_NO_CAST_TO_ASCII -DQT_ASCII_CAST_WARNINGS -DQT_MOC_COMPAT -DQT_USE_QSTRINGBUILDER -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -DQT_HAVE_MMX -DQT_HAVE_3DNOW -DQT_HAVE_SSE -DQT_HAVE_MMXEXT -DQT_HAVE_SSE2 -DQT_HAVE_SSE3 -DQT_HAVE_SSSE3 -DQT_HAVE_SSE4_1 -DQT_HAVE_SSE4_2 -DQT_HAVE_AVX -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE
 CFLAGS        = -pipe -fconstant-cfstrings -O2 -fvisibility=hidden -Wall -W -fPIC $(DEFINES)
 CXXFLAGS      = -I/Applications/GNURadio.app/Contents/MacOS/usr/include -I/opt/X11/include -std=c++11 -fconstant-cfstrings -O2 -fvisibility=hidden -fvisibility-inlines-hidden -Wall -W -fPIC $(DEFINES)
