diff --git a/src/corelib/global/qglobal.h b/src/corelib/global/qglobal.h
index 20d8832..d31ad57 100644
--- a/src/corelib/global/qglobal.h
+++ b/src/corelib/global/qglobal.h
@@ -991,6 +991,8 @@ namespace QT_NAMESPACE {}
      WIN16    - unsupported
 */
 
+#define Q_WS_X11
+
 #if defined(Q_OS_MSDOS)
 #  define Q_WS_WIN16
 #  error "Qt requires Win32 and does not work with Windows 3.x"
@@ -1032,6 +1034,10 @@ namespace QT_NAMESPACE {}
 #  define Q_WS_WIN
 #endif
 
+#undef Q_WS_MAC
+#undef Q_WS_MAC32
+#undef Q_WS_MAC64
+
 QT_BEGIN_HEADER
 QT_BEGIN_NAMESPACE
 
