diff --git a/mkspecs/darwin-g++/qmake.conf b/mkspecs/darwin-g++/qmake.conf
index f87a495..542dbf6 100644
--- a/mkspecs/darwin-g++/qmake.conf
+++ b/mkspecs/darwin-g++/qmake.conf
@@ -6,7 +6,7 @@
 
 MAKEFILE_GENERATOR	= UNIX
 TEMPLATE		= app
-CONFIG			+= qt warn_on release link_prl native_precompiled_headers
+CONFIG			+= qt warn_on release link_prl native_precompiled_headers lib_version_first
 QT			+= core gui
 DEFINES                 += __USE_WS_X11__
 
