commit 099085b09d4d55928297309c3b939188073e4ff4
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Sun Feb 12 22:18:17 2017 -0500

    add missing files

diff --git a/AUTHORS b/AUTHORS
new file mode 100644
index 0000000..8b13789
--- /dev/null
+++ b/AUTHORS
@@ -0,0 +1 @@
+
diff --git a/ChangeLog b/ChangeLog
new file mode 100644
index 0000000..8b13789
--- /dev/null
+++ b/ChangeLog
@@ -0,0 +1 @@
+
diff --git a/NEWS b/NEWS
new file mode 100644
index 0000000..8b13789
--- /dev/null
+++ b/NEWS
@@ -0,0 +1 @@
+
diff --git a/README b/README
new file mode 100644
index 0000000..8b13789
--- /dev/null
+++ b/README
@@ -0,0 +1 @@
+
