commit 1cb02b83930af42f54f65559573ddbfcafc301ea
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Wed Feb 15 23:14:38 2017 -0500

    LDFLAGS!!!!!

diff --git a/wxPython/setup.py b/wxPython/setup.py
index 8d3381e..f4f6962 100755
--- a/wxPython/setup.py
+++ b/wxPython/setup.py
@@ -180,6 +180,8 @@ else:
     rc_file = []
 
 
+extra_link_args = os.environ.get('LDFLAGS')
+
 ext = Extension('_core_', ['src/helpers.cpp',
                            ] + rc_file + swig_sources,
 
