commit e3115532905944d91feeb1ec0199b9f19146b438
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Sat Feb 11 09:17:31 2017 -0500

    make gtk work on mac

diff --git a/wxPython/config.py b/wxPython/config.py
index 08f8d6a..0e7e971 100644
--- a/wxPython/config.py
+++ b/wxPython/config.py
@@ -248,8 +248,8 @@ cleaning = 'clean' in sys.argv
 
 
 # change the PORT default for wxMac
-if sys.platform[:6] == "darwin":
-    WXPORT = 'osx_carbon'
+#if sys.platform[:6] == "darwin":
+#    WXPORT = 'osx_carbon'
 
 # and do the same for wxMSW, just for consistency
 if os.name == 'nt':
@@ -1051,13 +1051,14 @@ elif os.name == 'posix' or COMPILER == 'mingw32':
     if sys.platform[:6] == "darwin":
         WXPLAT = '__WXMAC__'
     
-        if WXPORT == 'osx_carbon':
-        # Flags and such for a Darwin (Max OS X) build of Python
-            GENDIR = 'osx_carbon'
-            WXPLAT2 = '__WXOSX_CARBON__'
-        else:
-            GENDIR = 'osx_cocoa'
-            WXPLAT2 = '__WXOSX_COCOA__'
+#        if WXPORT == 'osx_carbon':
+#        # Flags and such for a Darwin (Max OS X) build of Python
+#            GENDIR = 'osx_carbon'
+#            WXPLAT2 = '__WXOSX_CARBON__'
+#        else:
+#            GENDIR = 'osx_cocoa'
+#            WXPLAT2 = '__WXOSX_COCOA__'
+        GENDIR="gtk"
 
         libs = ['stdc++']
         NO_SCRIPTS = 1
