commit 148cf1804e5d6cf3dd21acd919ab48a5b8826441
Author: John Doe <jdoe@Johns-Mac.local>
Date:   Thu Feb 9 19:55:35 2017 -0500

    apply upstream fix that should allegedly fix macosx install

diff --git a/src/osx/carbon/dataobj.cpp b/src/osx/carbon/dataobj.cpp
index 758e3a7..5445aa6 100644
--- a/src/osx/carbon/dataobj.cpp
+++ b/src/osx/carbon/dataobj.cpp
@@ -29,10 +29,6 @@
 
 #include "wx/osx/private.h"
 
-#if wxOSX_USE_COCOA_OR_CARBON
-    #include <QuickTime/QuickTime.h>
-#endif
-
 // ----------------------------------------------------------------------------
 // wxDataFormat
 // ----------------------------------------------------------------------------
diff --git a/src/osx/core/bitmap.cpp b/src/osx/core/bitmap.cpp
index 3c61c17..3322b60 100644
--- a/src/osx/core/bitmap.cpp
+++ b/src/osx/core/bitmap.cpp
@@ -35,10 +35,6 @@ IMPLEMENT_DYNAMIC_CLASS(wxMask, wxObject)
 #include "wx/osx/private.h"
 #endif
 
-#ifndef __WXOSX_IPHONE__
-#include <QuickTime/QuickTime.h>
-#endif
-
 CGColorSpaceRef wxMacGetGenericRGBColorSpace();
 CGDataProviderRef wxMacCGDataProviderCreateWithMemoryBuffer( const wxMemoryBuffer& buf );
 
